MAINTAINERS.md

```diff
@@ -0,0 +1,15 @@
+The current Maintainers Group for the Kubevirt Project consists of:
+
+| Name | Employer | Responsibilities |
+| ----------- | ------- | -------------------------------- |
+| [Marc Sluiter](https://github.com/slintes) | Red Hat | |
+| [David Vossel](https://github.com/davidvossel) | Red Hat | |
+| [Vladik Romanovsky](https://github.com/vladikr) | Red Hat | |
+| [Roman Mohr](https://github.com/rmohr) | Red Hat | |
+| [Artyom Lukianov](https://github.com/cynepco3hahue) | Red Hat | |
+| [Sebastian Scheinkman](https://github.com/SchSeba) | Red Hat | |
+| [Marcin Franczyk](https://github.com/mfranczy) | Red Hat | |
+| [Fabian Deutsch](https://github.com/fabiand) | Red Hat | |
+| [Stu Gott](https://github.com/stu-gott) | Red Hat | |
+
```

Comment:
I'd like to propose to add Ryan Hallisey and potentially Marcus Sorensen to the initial list of contributors.

To me both of them are already meeting the maintainer requirements.

Reply:
@rthallisey @mlsorensen do you have thoughts on this?
Would you be willing to be a maintainer and take these responsibilites associated with it?

Reply:
+1 from me on these two as cncf maintainers for kubevirt

Reply:
As discussed in the community meeting I would first want to see a sign of life from @mlsorensen that he will be available and responsive if needed. Apart from that, both will work from me.

Reply:
We did not get a reply from Marcus and Ryan here, therefor I'm proposing to drop them.